### ISSUE TITLE
`PANIC: Invalid opcode at 0x3` in QEMU

### ISSUE THREAD

**Issue by enihcam** [Limine-Bootloader/Limine#269]
![image](https://user-images.githubusercontent.com/7010097/234653934-1716f6e6-6825-4798-93a9-45c27da93164.png)

after i added the following lines to limine.cfg and reboot
```
QUIET=yes
SERIAL=yes
GRAPHICS=no
EDITOR_ENABLED=no
...
:Arch Linux
  ...
  TEXTMODE=yes
```

**Comment by Mintsuki** [comment]
Sorry for the multiple commits. I recalled GitHub being able to automatically close PRs given the proper wording, but oh well...

In any case, I hope this commit fixes the issue. I'll close this issue once a release containing the fix is published.

**Comment by Mintsuki** [comment]
I forgot to mention that as a quick temporary workaround you can avoid using the `QUIET=yes` and `GRAPHICS=no` options at the same time.

**Comment by Mintsuki** [comment]
Should be fixed in 4.20230428.0.

Please feel free to reopen if that isn't the case.
